packages/console/src/components/CodeEditor/index.module.scss

```diff
@@ -48,7 +48,7 @@
       font-size: 14px;
       line-height: 1.5;
       font-family: 'Roboto Mono', monospace;
-      white-space: nowrap;
+      white-space: pre;
```

Comment:
didn't understand the PR description, what was the issue exactly?

Reply:
The code editor is combined with two parts: textarea component contains contents and syntax highlighter displays contents.
The issue is that, textarea component failed to wrap the line on newlines (written in contents).
w/o selecting:
<img width="987" alt="image" src="https://user-images.githubusercontent.com/15182327/228445478-a319d582-c064-42a9-b85d-d6f30dd9654e.png">
w/ content selected:
<img width="1033" alt="image" src="https://user-images.githubusercontent.com/15182327/228445450-96521e5e-9309-4415-95ac-1b8b006e7c45.png">